### PR TITLE
Three more pathfinders cards.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/pathfinders/Anthozoa.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      name: CardName.ANTHOZOA,
+      cost: 9,
+      tags: [Tags.PLANT, Tags.ANIMAL, Tags.MARS],
+      requirements: CardRequirements.builder((b) => b.oceans(3)),
+      resourceType: ResourceType.ANIMAL,
+
+      metadata: {
+        cardNumber: 'Pf55',
+        renderData: CardRenderer.builder((b) => {
+          b.action('Spend 1 plant to add an aimal to this card..', (eb) => {
```
Comment (bafolts) [inline src/cards/pathfinders/Anthozoa.ts:27]:
```suggestion
          b.action('Spend 1 plant to add an animal to this card..', (eb) => {
```

File: src/cards/pathfinders/Anthozoa.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    player.game.log('${0} spent 1 plant to place an animal on ${1}.', (b) => b.player(player).card(this));
+    return undefined;
+  }
+
+  public play() {
+    return undefined;
+  }
+
+  public getVictoryPoints() {
+    return Math.floor(this.resourceCount / 2);
```
Comment (bafolts) [inline src/cards/pathfinders/Anthozoa.ts:55]: This can use the new victory point format.

File: src/cards/pathfinders/Solarpedia.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  }
+
+  public play(player: Player) {
+    player.game.defer(new AddResourcesToCard(player, ResourceType.DATA, {count: 2}));
+    return undefined;
+  }
+
+
+  public getVictoryPoints() {
+    return Math.floor(this.resourceCount / 6);
```
Comment (bafolts) [inline src/cards/pathfinders/Solarpedia.ts:56]: This can use new victory point syntax.

**Review by bafolts — APPROVED** [review]

File: src/cards/pathfinders/Anthozoa.ts (hunk 2 above)
Comment (kberg) [reply src/cards/pathfinders/Anthozoa.ts:55]: Nice!

File: src/cards/pathfinders/Solarpedia.ts (hunk 3 above)
Comment (kberg) [reply src/cards/pathfinders/Solarpedia.ts:56]: woo

**Pull request merged**